app/tests/spec/models/page-refresh-watch.js

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'lib/metrics',
+  '../../mocks/window',
+  'models/notifications',
+  'models/page-refresh-watch',
+  'sinon'
+],
+function (chai, Metrics, WindowMock, Notifications, PageRefreshWatch, sinon) {
+  'use strict';
+
+  var assert = chai.assert;
+
+  describe('models/page-refresh-watch', function () {
+    var metrics;
+    var notifications;
+    var pageRefreshWatch;
+    var windowMock;
+
+    var ViewMock = function () {};
+
+    beforeEach(function () {
+      metrics = new Metrics();
+      sinon.spy(metrics, 'logScreenEvent');
+
+      notifications = new Notifications();
+      windowMock = new WindowMock();
+
+      pageRefreshWatch = new PageRefreshWatch({
+        metrics: metrics,
+        notifications: notifications,
+        window: windowMock
+      });
+    });
+
+    describe('logIfRefresh', function () {
+      describe('with two consecutive views with different names', function () {
+        it('does not log a page refresh', function () {
+          pageRefreshWatch.logIfRefresh('screen1');
+          pageRefreshWatch.logIfRefresh('screen2');
```

Comment:
This test and the following test contain identical boilerplate. Should be extracted to `before`.